rule-engine/rule-engine-components/src/test/java/org/thingsboard/rule/engine/action/TbLogNodeTest.java

```diff
@@ -77,4 +79,14 @@ void givenNullDataMsg_whenToLog_thenReturnString() {
                 "{}");
     }
 
+    @Test
+    void givenDefaultConfig_whenIsStandardForEachScriptLanguage_thenTrue() {
+        TbLogNode node = new TbLogNode();
+        for (ScriptLanguage scriptLanguage : ScriptLanguage.values()) {
+            TbLogNodeConfiguration config = new TbLogNodeConfiguration().defaultConfiguration();
+            config.setScriptLang(scriptLanguage);
+            assertThat(node.isStandard(config)).as("Script is standard for language " + scriptLanguage).isTrue();
```

Comment:
Instead of making it non-private, you may call init() with mock context and check that the ctx.createScriptEngine was never executed. This is the more correct way because we are interested in checking that we don't create the script engine for default scripts.

Reply:
done. with couple more checks
![image](https://user-images.githubusercontent.com/79898499/225005738-3c5727cb-f87d-4947-a208-328912ad60e0.png)